test/topology_experimental_raft/test_tablets.py

```diff
@@ -258,6 +258,22 @@ async def check():
     await cql.run_async("DROP KEYSPACE test;")
 
 
+@pytest.mark.asyncio
+async def test_mv_tablets_invalid_table(manager: ManagerClient):
+    """Scylla should return an HTTP error if the specified table is not found
+    """
+    servers = [await manager.server_add(),
+               await manager.server_add()]
+    src_host_id = await manager.get_host_id(servers[0].server_id)
+    dst_host_id = await manager.get_host_id(servers[1].server_id)
+    with pytest.raises(HTTPError, match="Can't find a column family"):
+        await manager.api.move_tablet(node_ip=servers[0].ip_addr,
```

Comment:
You could put it in test/topology, but there is no guarantee whether a cluster will be reused, depends on how the tests are ordered in the run, most likely this will be put somewhere between tests that are dirty-ing the cluster and there will be no reuse, and you'll lose because test/topology starts 3 nodes instead of 2.

I would:
- merge this with some other test case that checks invalid conditions
- or make it dev-mode or release-mode only (not sure if there's value in running this particular test case in debug)

Reply:
Maybe we should have a suite with such tests where test cases don't dirty the cluster.

Reply:
Why don't we have rest/test_api as well?

Reply:
moved this test into `moved it into `test/rest_api/test_storage_service.py`.